{
  "po" : 
  {
    "en_US" : 
    {
      "branch" : "",
      "commit" : 
      {
        "date" : "",
        "hash" : "",
        "title" : ""
      }
    },
    "zh_CN" : 
    {
      "branch" : "main",
      "commit" : 
      {
        "date" : "2025-09-15 02:11:01 -0400",
        "hash" : "481588ad4ccd2aff7ca3bac43379da0ce3b1c6d5",
        "title" : "gh-138871: Clarify NameError exception in 'del' (#138881)"
      }
    },
    "zh_TW" : 
    {
      "branch" : "main",
      "commit" : 
      {
        "date" : "2025-09-15 02:11:01 -0400",
        "hash" : "481588ad4ccd2aff7ca3bac43379da0ce3b1c6d5",
        "title" : "gh-138871: Clarify NameError exception in 'del' (#138881)"
      }
    }
  },
  "pot" : 
  {
    "branch" : "main",
    "commit" : 
    {
      "date" : "2025-09-15 02:11:01 -0400",
      "hash" : "481588ad4ccd2aff7ca3bac43379da0ce3b1c6d5",
      "title" : "gh-138871: Clarify NameError exception in 'del' (#138881)"
    }
  },
  "type" : "branch",
  "version" : "main"
}